blazor/circular-gauge/getting-started.md

```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 This section briefly explains about how to include [Blazor CircularGauge](https://www.syncfusion.com/blazor-components/blazor-circular-gauge) component in your Blazor Server App and Blazor WebAssembly App using Visual Studio.
 
+The below video demonstrates the customization options and features within the Blazor Circular Gauge component.
```

Comment:
The below video demonstrates the customization of Blazor Circular Gauge component.